skcosmo/preprocessing/FlexibleScaler.py

```diff
@@ -0,0 +1,220 @@
+import numpy as np
```

Comment:
Could you rename the file according to Python conventions, i.e. `flexible_scaler.py`?